src/scmdata/plotting.py

```diff
@@ -0,0 +1,107 @@
+"""
+Plotting helpers for DataFrames
+
+See the example notebook 'plotting-with-seaborn.ipynb' for examples on how to use
+"""
+
+import warnings
+
+import seaborn as sns
+
+CMIP6_SCENARIO_COLOURS = {
```

Comment:
```suggestion
RCMIP_SCENARIO_COLOURS = {
```

Ideal would be to auto-use https://github.com/IPCC-WG1/colormaps but that repo seems very inactive so this'll do :)